packages/server/src/modules/upload/resolvers.js

```diff
@@ -27,7 +35,7 @@ const storeFS = ({ stream, filename }) => {
 };
 
 const processUpload = async uploadPromise => {
-  const { stream, filename: name, mimetype: type, encoding } = await uploadPromise;
```

Comment:
For some reason `filename` and `mimetype` are undefined?

Reply:
@mitjade Maybe it should be vice versa, e.g. `name: filename`?

Reply:
I just removed the renaming and it works ok, just not sure why this is not working. If `uploadPromise` returns `name` and you want to assign it to `filename`?

Reply:
It have returned `filename` previously and I needed `name` from it. And it returned `mimetype` and I needed `type` instead to keep backward compatibility with old upload namings

Reply:
So is it ok now as is?

Reply:
I think yes